source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -839,6 +843,32 @@ export class CardDesignerSurface {
         }
     }
 
+    // Outstanding issue: cannot drop an element into a ColumnSet
+    // Find the index of the designer surface where the dragged peer should be dropped
+    calculateDraggedPeerIndex(currentRoot: DesignerPeers.DesignerPeer): boolean {
+        // Need to be able to access the index of the dragged peer in the container
+        if (this._draggedPeer instanceof DesignerPeers.CardElementPeer) {
+            for (let i = 0; i < currentRoot.getChildCount(); i++) {
```

Comment:
what is currentRoot?

Reply:
`currentRoot` is the element we are currently recursing through. It is initially the entire card, and as we come across additional containers, we count the elements in those as well.